newIDE/app/src/SceneEditor/LeaderboardDialog.js

```diff
@@ -0,0 +1,37 @@
+//@flow
```

Comment:
Should this file really be inside SceneEditor folder? It could be outside, maybe even inside a `src/Leaderboards` folder